packages/mask/src/extension/popups/pages/Swap/SwapBox/index.tsx

```diff
@@ -31,9 +27,5 @@ export function SwapBox() {
         history.replace(PopupRoutes.Swap)
     }, [chainId])
 
-    return (
-        <TradeContext.Provider value={tradeContext}>
-            <Trader coin={coin} chainId={chainId} />
-        </TradeContext.Provider>
-    )
```

Comment:
There are many hooks in Trader that depend on `TraderContext`. removing it will cause an error.

Reply:
> There are many hooks in Trader that depend on `TraderContext`. removing it will cause an error.

Since we combine all traders, we don't rely on a specify trader anymore.